website/docs/r/schema_site_bd.html.markdown

```diff
@@ -41,5 +43,5 @@ No attributes are exported.
 An existing MSO Schema Site Bridge Domain(BD) can be [imported][docs-import] into this resource via its Id/path, via the following command: [docs-import]: <https://www.terraform.io/docs/import/index.html>
 
 ```bash
-terraform import mso_schema_site_bd.bd1 {schema_id}/site/{site_id}/bd/{bd_name}
+terraform import mso_schema_site_bd.bd1 {schema_id}/site/{site_id}-{template_name}/bd/{bd_name}
```

Comment:
if decided to adjust to import statement logic also adjust documentation

Reply:
ok